homeassistant/components/trace/trace.py

```diff
@@ -0,0 +1,35 @@
+"""Support for automation and script tracing and debugging."""
```

Comment:
This file needs a better name. Any reason we wouldn't just put this in the `__init__`? We don't want other integrations to be aware of implementation details of the trace integration.

Reply:
Fixed in #48255